addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,25 +2,22 @@
-        <div t-on-mouseenter="this.props.freeze" t-on-mouseleave="this.props.refresh" t-attf-class="o_notification {{props.className}} border border-{{props.type}} bg-view mb-2 position-relative"
-         role="alert" aria-live="assertive" aria-atomic="true">
-            <strong t-if="props.title" t-attf-class="o_notification_title d-block text-{{props.type}} py-2 ps-3 pe-5" t-out="props.title"/>
-            <button type="button" class="o_notification_close btn" aria-label="Close" t-on-click="props.close">
-                <i class="oi oi-close"/>
-            </button>
-            <div t-attf-class="o_notification_body ps-3 pe-5 py-2">
-                <div t-if="props.message" class="me-auto o_notification_content" t-out="props.message"/>
-                <div t-if="props.buttons.length" class="o_notification_buttons d-flex gap-2 mt-2">
-                    <button t-foreach="props.buttons" t-as="button" type="button" t-key="button_index"
-                    t-attf-class="btn {{button.primary ? 'btn-primary' : 'btn-secondary'}}" t-on-click="button.onClick">
-                    <t t-if="button.icon">
-                        <i t-if="button.icon.indexOf('fa-') === 0" role="img"
-                        t-att-aria-label="button.name" t-att-title="button.name"
-                        t-attf-class="fa fa-fw o_button_icon {{button.icon}}"/>
-                        <img t-else="" t-att-src="button.icon" t-att-alt="button.name"/>
-                    </t>
-                    <span t-esc="button.name"/>
-                </button>
+        <div t-on-mouseenter="this.props.freeze" t-on-mouseleave="this.props.refresh" t-attf-class="o_notification {{props.className}} d-flex mb-2 position-relative rounded bg-view shadow-lg" role="alert" aria-live="assertive" aria-atomic="true">
+            <span t-attf-class="o_notification_border bg-{{props.type}} rounded-start"/>
+            <div class="w-100 py-3 ps-3 pe-5 border border-start-0 rounded-end">
+                <h5 t-if="props.title" class="o_notification_title m-0" t-out="props.title"/>
+                <button type="button" class="o_notification_close btn-close position-absolute top-0 end-0 mt-3 me-3" aria-label="Close" t-on-click="props.close"/>
+                <div class="o_notification_body pe-5" t-att-class="{'mt-2' : props.title}">
+                    <div t-if="props.message" class="me-auto o_notification_content" t-out="props.message"/>
+                    <div t-if="props.buttons.length" class="o_notification_buttons d-flex gap-2 mt-3">
+                        <button t-foreach="props.buttons" t-as="button" type="button" t-key="button_index" t-attf-class="btn {{button.primary ? 'btn-primary' : 'btn-secondary'}}" t-on-click="button.onClick">
+                            <t t-if="button.icon">
+                                <i t-if="button.icon.indexOf('fa-') === 0" role="img" t-att-aria-label="button.name" t-att-title="button.name" t-attf-class="fa fa-fw o_button_icon"/>
```

Comment:
`{{button.icon}}` is missing thus no icon is rendered ;)
In a previous comment I said that `o_button_icon` class is deprecated and could be removed, maybe you confused the two?

Reply:
Indeed I mingled both 
Added a margin class as a counterpart